lib/sinon/assert.js

```diff
@@ -78,6 +78,11 @@ function mirrorPropAsAssertion(name, method, message) {
         var args = slice.call(arguments, 1);
         var failed = false;
 
+        // Hack to be replaced
```

Comment:
This seems like a TODO statement. No one ever does anything to a TODO statement. Could you rather expand on your comment? I am not totally sure what this does.